yarpl/include/yarpl/Refcounted.h

```diff
@@ -176,35 +181,124 @@ bool operator==(const Reference<T>& lhs, const Reference<U>& rhs) noexcept {
   return lhs.get() == rhs.get();
 }
 
+template <typename T>
+bool operator==(const Reference<T>& lhs, std::nullptr_t) noexcept {
```

Comment:
When I check folly/io/DelayedDesctructionBase.h file, which is similar to this implementation, I see that it is providing these functions also. Though it doesn't provide implicit constructor with std::nullptr_t.
As there is implicit constructor, it seems like these methods are not needed..